recipe/meta.yaml

```
@@ -25,7 +26,7 @@ source:
     sha256: 5daca6ca216495edf89d167f808d1d03c4a4d929cef7da5e10f135ae1540c7e4
     folder: json
 build:
-  number: 1
+  number: 3
   skip: true  # [win]
   # Since 1.11, power9 seems to be required.
   skip: true  # [ppc64le]
```

You should also ignore the generated `run_exports` to make this compatible with other versions of dynamically linked protobuf.
```suggestion
  skip: true  # [ppc64le]
  ignore_run_exports_from:
    - libprotobuf
```

---

Thanks, done!